pkg/selectors/kernel.go

```diff
@@ -200,6 +200,9 @@ const (
 	// more socket ops
 	SelectorOpFamily = 28
 	SelectorOpState  = 29
+
+	// capabilities mask
+	SelectorOpCapability = 30
```

Comment:
just curious.. is there any point in having negative cap filter as well? to filter all the calls that are not CAP_ADMIN or such?

Reply:
@olsajiri, sorry for the ping here, but do you have any recommendation on how we could add this new filter in the tailcal without restricting it to LARGE_BPF_PROG ? Or it does not matter if Capability is available for all the kernel versions but NotCapability only for 5.4+ ?

Reply:
I guess it's fine.. and it's also fine if we do that as a follow up when we see that it's really needed, I dont insist ;-)

Reply:
Oh, ok ! So this PR is good at this point.
Thank you for the reviews 💯